### PR TITLE
fix: Add limit for token TTL upper bound

### PR DESCRIPTION
**Pull request by sailorlqh**
Fix issue: https://github.com/abcxyz/jvs/issues/169

### CODE REVIEW

File: pkg/justification/processor_test.go [hunk 1]
```diff
@@ -109,6 +109,13 @@ func TestCreateToken(t *testing.T) {
 			},
 			wantErr: "failed to validate request",
 		},
+		{
+			name: "ttl_too_long",
+			request: &jvspb.CreateJustificationRequest{
+				Ttl: durationpb.New(25 * time.Hour),
+			},
+			wantErr: "failed to validate request",
```
Comment (yolocs) [inline pkg/justification/processor_test.go:117]: Should we use the new error message?

File: pkg/justification/processor.go [hunk 2]
```diff
@@ -164,7 +164,11 @@ func (p *Processor) createToken(ctx context.Context, now time.Time, req *jvspb.C
 	}
 
 	id := uuid.New().String()
+	if req.Ttl.AsDuration() > 24*time.Hour {
+		return nil, fmt.Errorf("token ttl should be less than 24 hours")
+	}
```
Comment (yolocs) [inline pkg/justification/processor.go:169]: Relocate to `runValidations`?

**Review by yolocs — CHANGES_REQUESTED** [review]

File: pkg/justification/processor_test.go (hunk 1 above)
Comment (capri-xiyue) [reply pkg/justification/processor_test.go:117]: +1 can we add more specific wantErr substring in UT?

File: pkg/justification/processor.go [hunk 3]
```diff
@@ -78,8 +78,9 @@ func (p *Processor) CreateToken(ctx context.Context, req *jvspb.CreateJustificat
 	logger := logging.FromContext(ctx)
 
 	if err := p.runValidations(req); err != nil {
+		// panic(err.Error())
 		logger.Errorw("failed to validate request", "error", err)
-		return nil, status.Error(codes.InvalidArgument, "failed to validate request")
+		return nil, status.Error(codes.InvalidArgument, "failed to validate request, "+err.Error())
```
Comment (capri-xiyue) [inline pkg/justification/processor.go:83]: use`fmt.Sprintf()`

**Comment by sethvargo** [comment]
Sorry hang on. I have a PR in progress for this, but was trying to land a few other things first. I think we want to do something a bit more sophisticated which I have coded up locally, but was running into issues testing (which Chen fixed in another PR).

**Comment by sethvargo** [comment]
https://github.com/abcxyz/jvs/pull/176

**Pull request closed without merging**